### PR TITLE
[ENH, MRG] Add dependencies to tutorials, fix GUI sizing

### PR DESCRIPTION
**Pull request by alexrockhill**
Fixes review suggestions in https://github.com/openjournals/joss-reviews/issues/3897.

### CODE REVIEW

**Review by agramfort — APPROVED** [review]
+1 for merge when CIs are green

Thx @alexrockhill

**Review by drammock — APPROVED** [review]
changes look reasonable.  Some nitpicks

1. in the ieeg_localize tutorial, in the "aligning to ACPC" section, use the sphinx `:menuselection:` role for the freeview commands (see here https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-menuselection)
2. in the "marking the location" section under "to operate the GUI" there's an unnecessary indentation that makes the bullet list and subsequent 3 notes boxes look like they're all part of a blockquote
3. final image of that tutorial shows some electrodes outside the brain, is that expected?

**Comment by alexrockhill** [comment]
Looks like the test failures are an unrelated deprecation warning from `nibabel`.

File: tutorials/clinical/10_ieeg_localize.py [hunk 1]
```diff
@@ -207,7 +207,8 @@ def plot_overlay(image, compare, title, thresh=None):
 #         - Load the two scans from the command line using
 #           ``freeview $MISC_PATH/seeg/sample_seeg/mri/T1.mgz
 #           $MISC_PATH/seeg/sample_seeg_CT.mgz``
-#         - Navigate to the upper toolbar, go to ``Tools>>Transform Volume...``
+#         - Navigate to the upper toolbar, go to
+#           :menuselection:`Tools --> Transform Volume...`
```
Comment (drammock) [inline tutorials/clinical/10_ieeg_localize.py:211]: why only this one?  There is a `Tools -> transform volume` above as well as `Freeview -> Preferences -> General -> Cursor style -> Long` (again, both in the "aligning to ACPC" section)

**Comment by alexrockhill** [comment]
Ok, looked good locally to me and passes all the tests!

**Comment by alexrockhill** [comment]
Looks good on the artifact too https://40002-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/clinical/10_ieeg_localize.html

**Pull request merged**